### ISSUE TITLE
Expo Local Authentication TouchID/FaceID not working in multiple IPA with "error":"unknown: -1004, User interaction is required." 

### ISSUE THREAD

**Issue by affcue25** [expo/expo#10200]
I have 2 app that are using expo local authentication. When I install both of them on iphone then TouchID/FaceID stop triggering in the first installed app. But works in the last installed app. 
The same thing working in android perfectly for both apps. Even I have different meta data for both apps in app.json 

**1st app app.json meta data for ios**
`"ios": {`
      `"supportsTablet": true,`
      `"bundleIdentifier": "com.aivatek.onquest",`
      `"infoPlist": {`
        `"NSCameraUsageDescription": "This app uses the camera to scan barcodes on event tickets.",`
        `"NSFaceIDUsageDescription":"This app uses the camera to scan Face.",`
        `"CFBundleExecutable": "ComAivatekOnquest"`
     ` }`
    `},`

**2nd app app.json meta data for ios**
`"ios": {`
      `"supportsTablet": true,`
     ` "bundleIdentifier": "com.aivatek.onquestcaptain",`
      `"infoPlist": {`
       ` "NSCameraUsageDescription": "This app uses the camera to scan barcodes on event tickets.",`
        `"NSFaceIDUsageDescription": "This app uses the camera to scan FaceID.",`
        `"CFBundleExecutable": "ComAivatekOnquestcaptain"`
      `}`
   ` },`

**Error I get:** 
"error":"unknown: -1004, User interaction is required." 

Can anyone tell me whats wrong with it?

**Comment by affcue25** [comment]
Okay finally I solved it!!!
My both apps were using expo sdk 38 and react-native sdk 38.
I just downgraded only one app to

```
“expo”: “^36.0.0”,
“react-native”: “https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz”
```

the other app still using sdk 39. and then I removed the node_modules and package-lock.json from the downgraded app.

`sudo rm -rf ./node_modules && sudo rm -rf package-lock.json
`
and then I Installed new modules in it

`npm i`

solved some compatible warning of libraries by doing

`expo install “Library name”`

Built the app, and installed both apps and it worked!!. :+1:
